rules/cross-platform/discovery_security_software_grep.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2020/12/20"
+maturity = "production"
+updated_date = "2020/12/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies the use of the grep command to discover known third-party macOS and Linux security tools, such as
+AntiVirus or Host Firewall.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "auditbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Security Software Discovery using Grep"
+risk_score = 47
+rule_id = "870aecc0-cea4-4110-af3f-e02e9b373655"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Linux", "Threat Detection", "Discovery"]
+type = "query"
+
+query = '''
+event.category : process and event.type : (start or process_started) and
```

Comment:
consider switching the query to EQL to take advantage of case insensitivity, i.e, -
```
process.args: 
  (
  "Little Snitch",
  "Avast*",
  "Avira*",
  "ESET*",
  etc...
  )
```

Reply:
case sensitivity not needed

Reply:
I'll disagree on that one. I think process.args should be case insensitive.  If an attacker executed `ps -ef | grep Elastic-Endpoint`, then it wouldn't trigger `elastic-endpoint` in the rule.

Reply:
there are tons of ways to bypass this detection, this is a light one, defenders can extend as they which.